docs/virtual_machines/disks_and_volumes.md

```diff
@@ -74,6 +74,11 @@ configuration:
     -  PersistentReservation
 ```
 
+The persistent reservation feature enables an additional privileged component to
```

Comment:
What about the following markup:

> [!WARNING]
> warn text

Reply:
This makes me wonder: How can an admin limit the use of this feature inside a cluster? aka limit it to a usr or to certain vms?

generally speaking something like kyverno could be used.

Reply:
Either you have persistent reservation enable at cluster level or not. My suggestion is to have it disable is because it deploys an additional privileged component, and this feature is quite niece. I wouldn't enable always by default.
In order to use SCSI persistent reservation, you also need to have a CSI plugin which support SCSI protocol. This feature requires in any case some work from the cluster-adin.

However, from regular user, if this is enable, I don't think we should limit this only to certain namespaces.

I hope this makes the entire feature a bit clearer. Please let me know, which aspect you would like me to capture in the documentation.

Reply:
An [admonition without a title](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#removing-the-title) (or a note admonition tbh) might be useful to bring attention to this para. It does seem worthy, and in its position it is a bit easy to miss. 
```
!!! note ""

    yada yada
```

Reply:
@alicefr Sorry to be a pain but the note text needs 4 spaces of indentation to render. 
It's currently rendering a single line: https://deploy-preview-772--kubevirt-user-guide.netlify.app/virtual_machines/disks_and_volumes/#persistent-reservation

Reply:
@aburdenthehand sorry, I used the > as we already have in the documentation